validation/rally-beyond-the-paywall/1.i.advertisements.pass.json

```diff
@@ -0,0 +1,34 @@
+{
+      "type": "WebScience.advertisements",
+      "userId": "b4aeab86-1535-4701-8a20-a9aab64e0866",
+      "visitId": "e44f4b1b-91b1-4a5f-ac22-82346d37c44a",
+      "url": "https://www.nytimes.com/2021/06/08/us/politics/capitol-riot-security.html",
+      "body": {
+        "clientHeight": 983,
+        "clientWidth": 1265
+      },
+      "ads": [
+        {
+          "id": "bottom",
+          "tag": "DIV",
+          "clientHeight": 90,
+          "clientWidth": 1265
+        },
+        {
+          "id": "google_ads_iframe_/29390238/nyt/us/politics_0",
+          "tag": "IFRAME",
+          "clientHeight": 250,
+          "clientWidth": 970,
+          "height": "250",
+          "width": "970"
```

Comment:
What prompted the collection for this string height/width field?

Reply:
The value is a strings sometimes.  Example:
"height": "350px",
"width": "300px"

https://code.stanford.edu/gjmartin/beyond_the_paywall/-/blob/master/schemas/advertisement-example.json